### PR TITLE
Hotfix #578

### LINKED ISSUE #578: Avoid ExploreASL searching in directories higher up
### Description

When starting ExploreASL with a certain path, it will also start looking in paths above it for `/rawdata` folders or `dataPar.json`. This creates the danger of starting to import or process a dataset that was already done. See printscreen below.

### Requirements

ExploreASL should only go down, not up, for trying to find the correct folder.

### Release notes

Avoid ExploreASL searching in directories higher up than the specified folder.

### PR DESCRIPTION
**Pull request by HenkMutsaerts**
### Linked issue

Closes #578 and to create release 1.6.1

### CODE REVIEW

**Review by MDijsselhof — APPROVED** [review]

**Review by BeatrizPadrela — APPROVED** [review]
seems ok!

**Review by jan-petr — APPROVED** [review]
Ready to release...

**Comment by jan-petr** [comment]
Sorry - my last commit wasn't signed... Will fix that.

**Review by MichaelStritt — CHANGES_REQUESTED** [review]
### GE_PCASL_3Dspiral_volunteer

```matlab
>> source = '.\New_TestData\GE_PCASL_3Dspiral_volunteer\sourceStructure.json';
>> [x] = ExploreASL_Master(source,1,1);

...

ExploreASL will run the import workflow and will run the processing pipeline...


==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
DataParPath         ...rver_xASL\New_TestData\GE_PCASL_3Dspiral_volunteer\sourceStructure.json
Import Modules      DCM2NII NII2BIDS ANONYMIZE BIDS2LEGACY
Process Modules     Structural ASL Population
bPause              False
iWorker             1
nWorkers            1
==============================================================================================
ExploreASL v1.6.1 initialized ... 

...

K>> x.DataParPath

ans =

    '.\New_TestData\GE_PCASL_3Dspiral_volunteer\sourceStructure.json'

Error using xASL_imp_BIDS2Legacy (line 30)
Input argument should be dataset_description.json, invalid input

Error in ExploreASL_ImportMaster (line 43)
                    x = xASL_imp_BIDS2Legacy(x);

Error in ExploreASL_Master (line 74)
    x = ExploreASL_ImportMaster(x);
```

File: Modules/SubModule_Import/xASL_imp_BIDS2Legacy.m [hunk 1]
```diff
@@ -23,37 +23,72 @@
...
-                dataPar = xASL_bids_BIDS2Legacy(thisRootFolder, 1, []);
-            else
-                % Fill the dataPars with the provided parameters
-                dataPar = spm_jsonread(fListDataPar{1});
-                dataPar = xASL_bids_BIDS2Legacy(thisRootFolder, 1, dataPar);
-            end
-        end
+
+    %% 1. Start with checking dataset_description.json & rawdata
+    [Fpath, Ffile, Fext] = fileparts(x.DataParPath);
```
[the remark this answers is not in the document]
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_BIDS2Legacy.m:28]: Check out my changes in b4706bd

[the remark this answers is not in the document]
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_BIDS2Legacy.m:28]:
```
================================= TEST RESULTS =================================

              name                  unit          tests        passed
    _________________________    __________    ____________    ______

    'xASL_bids_BIDS2Legacy'      'Function'    [1×2 struct]    true  
    'ExploreASL_Master'          'Function'    [1×7 struct]    true  
    'xASL_io_ExportVTK'          'Function'    [1×5 struct]    true  
    'xASL_test_GetLogContent'    'Function'    [1×2 struct]    true  
    'xASL_io_Nifti2Im'           'Function'    [1×1 struct]    true  
    'xASL_tsvRead'               'Function'    [1×2 struct]    true  
    'xASL_tsvWrite'              'Function'    [1×2 struct]    true  

================================================================================
```

[the remark this answers is not in the document]
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_BIDS2Legacy.m:28]: ## GE Datasets FlavorDatabase

[logContent.xlsx](https://github.com/ExploreASL/ExploreASL/files/6493551/logContent.xlsx)

### Check the BIDS conversion
`Different value: BIDSVersion (1.6.0 vs 1.5.0)`

**Review by MichaelStritt — APPROVED** [review]
I only re-tested GE and the UnitTests. All of that seems fine for me. 👍

**Comment by MichaelStritt** [comment]
### GE_PCASL_3Dspiral_volunteer

Re-tested, works fine 👍

**Pull request merged**